src/transformState.js

```diff
@@ -1,11 +1,63 @@
+/* eslint-disable spaced-comment */
 'use strict';
 
 /**
  * @param {Object} state
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const reducer = (_state, action) => {
```

Comment:
why do you use `_` for `state`?
It's a normal argument for the faction, don't use `_`

Reply:
@Denchepornyuk Its a "no-shadowing" rule in ESlinter